src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -0,0 +1,197 @@
+import { Tooltip } from 'antd'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+import Loading from 'components/shared/Loading'
+import ProjectLogo from 'components/shared/ProjectLogo'
+
+import { ThemeContext } from 'contexts/themeContext'
+import { BigNumber } from 'ethers'
+import { useProjectMetadata } from 'hooks/ProjectMetadata'
+import { CSSProperties, useContext } from 'react'
+import { formatWad } from 'utils/formatNumber'
+import { getTerminalVersion } from 'utils/terminal-versions'
+
+import { Project } from 'models/subgraph-entities/project'
+
+import { CURRENCY_ETH } from 'constants/currency'
+import { trendingWindowDays } from 'constants/trending-projects'
+
+export default function TrendingProjectCard({
+  project,
+  size,
+  bg,
+  rank,
+}: {
+  project: Project & { trendingVolume: BigNumber | undefined }
+  size?: 'sm' | 'lg'
+  bg?: string // Used on homepage
+  rank: number
+}) {
+  const {
+    theme: { colors, radii },
+  } = useContext(ThemeContext)
+
+  const cardStyle: CSSProperties = {
+    display: 'flex',
+    alignItems: 'center',
+    whiteSpace: 'pre',
+    overflow: 'hidden',
+    padding: '25px 20px',
+    backgroundColor: bg,
+    // Shows darker border when background is set
+    border: `1px solid ${
+      bg ? 'var(--stroke-secondary)' : 'var(--stroke-tertiary)'
+    }`,
+  }
+
+  const rankStyle: CSSProperties = {
+    fontSize: 22,
+    color: 'var(--text-primary)',
+    fontWeight: 400,
+    marginRight: 15,
+  }
+
+  const { data: metadata } = useProjectMetadata(project?.uri)
+
+  const terminalVersion = getTerminalVersion(project?.terminal)
+
+  // If the total paid is greater than 0, but less than 10 ETH, show two decimal places.
+  const precision =
+    project.trendingVolume?.gt(0) &&
+    project.trendingVolume.lt(BigNumber.from('10000000000000000000'))
```

Comment:
**suggestion**: I think this works:

https://docs.ethers.io/v5/api/utils/constants/#constants-WeiPerEther

```suggestion
    project.trendingVolume.lt(constants.WeiPerEther)
```